### DISCUSSION TITLE
Transaction Amount Sign Flip/Multiplier

### DISCUSSION THREAD

**Discussion by abishalom** [firefly-iii/firefly-iii#7782] [Firefly III] [answered]
Hi,

Some of my credit card accounts have different signs (i.e. spends are positive vs. negative). Is there a way to define a sign flip in the config file (or more abstractly some mult on the "amount" column) so that I don't have to manually edit the CSVs each time I want to import?

**Comment by JC5** [comment, marked as answer]
Yes, select "Amount (negated)" in the drop-down.
